.gitignore

```diff
@@ -35,7 +35,6 @@ dist
 docs/_build
 docs/source/apidocs
 obj/
-tests/resources/conformance_suites/
```

Comment:
This seems like it could cause one of the conformance suites which isn't public to inadvertently be committed and pushed. Can we use a [negation pattern](https://git-scm.com/docs/gitignore#_pattern_format) (`!` prefix) to include nl, but continue to exclude all others?